src/structures/Message.js

```diff
@@ -129,6 +129,12 @@ class Message extends Base {
          */
         this.fromMe = data.id.fromMe;
 
+        /**
+         * Indicates if the message was sent via whatsapp web
+         * @type {boolean}
+         */
+        this.isWWebJS = data.id.id.startsWith('3EB0') && data.id.id.length === 12;
```

Comment:
What about [`this.deviceType`](https://github.com/pedroslopez/whatsapp-web.js/blob/8f5d9ad6fce6a8e8746176892adc0b796d96e1d6/src/structures/Message.js#L93)?

Reply:
`this.deviceType` is only to indicate the type of device that is connected to whatsapp web, this one's function is to distinguish which messages are sent automatically and which messages are sent manually in whatsapp web